src/App.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import Books from './components/Books';
+import MenuLink from './components/MenuLink';
+import Categories from './routes/Categories';
+import Book from './routes/BookPage';
+import './styles/App.css';
+
+function App() {
+  return (
+    <div className="wrapper">
+      <Router>
+        <MenuLink />
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="/BookPage" element={<Book />} />
```

Comment:
- [ ] Please make sure that the book route renders the root directory that is it should have path  '/' . This way when the page loads it will be directly on the book component. Example snippet:

```js
  <Route path="/" element={<Books />} />
```